Makefile

```diff
@@ -56,6 +60,16 @@ SHELLCHECK_VERSION ?= "v0.10.0"
 GOLANG_CI_LINT = $(LOCALBIN)/golangci-lint
 GOLANG_CI_LINT_VERSION ?= "v2.1.5"
 
+.PHONY: tomljson
+tomljson: $(TOMLJSON)
+$(TOMLJSON): $(LOCALBIN)
+	$(call go-install-tool, $(TOMLJSON),github.com/pelletier/go-toml/v2/cmd/tomljson,$(GO_TOML_VERSION))
+
+.PHONY: jsontoml
+jsontoml: $(JSONTOML)
+$(JSONTOML): $(LOCALBIN)
+	$(call go-install-tool, $(JSONTOML),github.com/pelletier/go-toml/v2/cmd/jsontoml,$(GO_TOML_VERSION))
+
```

Comment:
NOTE: I would prefer using `go.mod` for most go binaries (see https://go.dev/ref/mod#go-mod-file-tool). This way, the tools can be checked and updated by dependabot.